data/human/ships.txt

```diff
@@ -3161,3 +3161,110 @@ ship "Wasp"
 	explode "tiny explosion" 20
 	explode "small explosion" 10
 	description "The Wasp is a medium-strength interceptor, designed by Syndicated Shipyards to serve as an escort for cargo vessels. Like most Syndicate ships, it is not particularly pretty, but it at least serves the purpose it was made for."
+
+ship "Quicksilver (FF)"
```

Comment:
I think these should go to variants, not human ships. However they seem to have different stats from the regular versions of the ships so I don't know what's going on with them.

Reply:
I took out cargo and put in more fuel capacity for the long distance nature of the missions. I plan on doing more missions where the player moves through the empty spaces of the Far North. I dont know the rules on changing things like this. I took like 10 cargo for each 100 fuel i added so it wouldn't be an unfair advantage.

Reply:
Should i not mess with the specs and just move around outfits? And move it into variants?

Reply:
It'd be better if they were ship variants instead of new ships. You can increase fuel capacity and decrease available outfit space by adding a Fuel Pod, which takes up 8 units of outfit space and adds 100 units of fuel (one jump of a standard hyperdrive). If the ships don't have enough outfit space for all the fuel pods you'd like, you can add outfit expansions, which convert cargo space into outfit space (and make it slightly less heat efficient).

Reply:
I got the changes done but i cant figure out how to give the ships static names. It looks like i need to use variant with fleet. The only workaround i see is making a phrase in the names.txt with just the names I want. Otherwise the names will change randomly from mission to mission.

Reply:
I tried directing names to a phrase with only two unique names in it for the two ships in the mission. But sometimes it gives both ships the same name. Is there a better way to do this? Or is it not a big deal that the same ships you're escorting along various missions keep changing names.

Reply:
I made variants and removed ships. Everything seems to be working now.